### PR TITLE
Renamed Firecracker master branch to *main*

### PR DESCRIPTION
**Pull request by luminitavoicu**
Signed-off-by: Luminita Voicu <lumivo@amazon.com>

*Issue #, if available:*

*Description of changes:*
Renamed Firecracker master branch to *main*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by samuelkarp — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ to report problems, discuss roadmap items, or make feature requests.
 If you've discovered an issue that may have security implications to
 users or developers of this software, please do not report it using
 GitHub issues, but instead follow
-[Firecracker's security reporting guidelines](https://github.com/firecracker-microvm/firecracker/blob/master/SECURITY-POLICY.md).
+[Firecracker's security reporting guidelines](https://github.com/firecracker-microvm/firecracker/blob/main/SECURITY-POLICY.md).
```
Comment (samuelkarp) [inline README.md:137]: Would you mind updating the link to reflect the file rename too?  It looks like the file got renamed to `SECURITY.md`.

```suggestion
[Firecracker's security reporting guidelines](https://github.com/firecracker-microvm/firecracker/blob/main/SECURITY.md).
```

(the above link doesn't work; I assume that the branch rename has not completed yet?)

Comment (kzys) [reply README.md:137]: I think so. Let's rename the branch on Firecracker first. So that we can merge this change.

Comment (luminitavoicu) [reply README.md]: @samuelkarp, you are right, the branch rename has not been completed yet, but will soon. I updated the PR to include the file name change you pointed out.

**Comment by luminitavoicu** [comment]
The default branch has been renamed. The link is working now.

**Review by samuelkarp — APPROVED** [review]

**Pull request merged**